qap.go

```diff
@@ -10,24 +10,24 @@ var instances = []string{"tai256c", "tho150", "wil50", "sko100c", "lipa80a", "nu
 
 //var instanceSizes = []int{256,150,50,100,80,30,20,32,12,26}
 
-const defaultSize = 256
+const defaultSize = 30
 const neighbourCount = defaultSize * (defaultSize - 1) / 2
 
-func steepest(assignment Assignment, m1 IntMat, m2 IntMat) (Assignment, int, int, int64) {
+func steepest(assignment Assignment, m1 IntMat, m2 IntMat) (Assignment, int, int, int, int64) {
 	start := time.Now()
 	currentAssignment := assignment
-	var bestCost, bestNeighbourCost, bestNeighbourIndex, stepCount int
+	var bestCost, bestNeighbourCost, bestNeighbourIndex, stepCount, exploredSolutions int
 	var costMatrix IntMat
 	for ok := true; ok; ok = bestNeighbourCost < bestCost {
 		bestCost, costMatrix = calcCost(currentAssignment, m1, m2)
 		neighbours, neighboursCosts := createNeighbours(currentAssignment, m1, m2, costMatrix, bestCost, rand.Intn(defaultSize))
+		exploredSolutions += neighbourCount - 1
```

Comment:
skąd -1 (domyślam się, że powód ten sam, co tam gdzie "nie czaję")

Reply:
pamiętam, że to zrobiłem i miałem jakiś powód, ale za nic sobie nie mogę przypomnieć co to było, bo to z tego porzedniego PR co wisi pare tygodni